common/src/main/java/com/wynntils/models/wynnalphabet/WynnAlphabetModel.java

```diff
@@ -351,21 +343,6 @@ private boolean hasTranscriber(WynnAlphabet transciberToFind) {
         return false;
     }
 
-    private boolean hasCompletedDiscovery(WynnAlphabet discoveryToCheck) {
```

Comment:
I think it is a bit sad to see that option go. I can say that I have never managed to complete any of these discoveries, and I've actually played with this setting on to kind of keep a bit of the suspense.

Can't we move this check to the character model instead, so we query the discoveries when the player enters a new world? Then it would be quick to check for this condition. I think we try to parse the quest book at that time anyhow, right?

Reply:
I think(?) we only scan the content book when the user opens the relevant screen, so discoveries would only be checked when the discoveries screen is opened. It could be done every world change but given the amount of discoveries it wouldn't be quick and I imagine we'd get complaints again about the inventory being replaced by the content book

Reply:
I think that basically brings me back to my old idea of essentially caching the quest book locally, so we track any changes (and has a full-rescan fallback in case of issues). Having the quest book data instantaneuosly available without cost would probably open up a lot of cool stuff, such as automatically indicating when you are near a starting point for a quest that you have not done, etc.

Reply:
So I agree, let's keep this removed for now, but if we do add the content book cache, we should re-implement this.

Reply:
We'll have to figure out a way to have time to do the query, we already do quite a lot on class join. But otherwise, great idea.